api/sds.proto

```diff
@@ -130,6 +140,8 @@ message DownstreamTlsContext {
 
   // If specified, Envoy will reject connections without a valid and matching SNI.
   google.protobuf.BoolValue require_sni = 3;
+
+  TlsSessionTicketKeys tls_session_ticket_keys = 4;
```

Comment:
Should I have a SdsSecretConfig here paired with this?  I don't grok the relationship between the inline-secrets vs the pointer-to-SDS secrets.

Reply:
Maybe we can add a `oneof` for new users @PiotrSikora ?

Reply:
Yeah, I was looking at this to figure out what to do:
    repeated TlsCertificate tls_certificates = 2;
    repeated SdsSecretConfig sds_secret_configs = 6;

Is that two separate fields instead of oneof for backwards compatibility reasons?  If so, should I add a comment to that effect?